Xamarin.Forms.Core/Switch.cs

```diff
@@ -39,7 +43,16 @@ public Switch()
 		public bool IsToggled
 		{
 			get { return (bool)GetValue(IsToggledProperty); }
-			set { SetValue(IsToggledProperty, value); }
+			set { SetValue(IsToggledProperty, value); ChangeVisualState(); }
```

Comment:
nope. the ChangeVisualState will be triggered by the property changed handler

Reply:
I've a question, What if someone not using the MVVM approach or Binding Property (IsToggledProperty) and use this public property IsToggled to set the value (though I know they need to have their own mechanism to update the UI)? 
It is similar to what we have with Checkbox: https://github.com/xamarin/Xamarin.Forms/blob/master/Xamarin.Forms.Core/CheckBox.cs#L38